src/browser_action/on-each-interaction.js

```diff
@@ -60,16 +60,11 @@ export function onEachInteraction(callback) {
     for (const interaction of Object.values(interactions)) {
       const entry = interaction.reduce((prev, curr) => prev.duration >= curr.duration ? prev : curr);
       const value = entry.duration;
-      const interactionId = entry.interactionId;
 
```

Comment:
Let's keep this in. Will explain in a minute why.

```suggestion
const interactionId = entry.interactionId;

```